packages/server/email/MailManager.ts

```diff
@@ -15,4 +17,7 @@ export interface MailManagerOptions {
 }
 export default abstract class MailManager {
   abstract sendEmail(options: MailManagerOptions): Promise<boolean>
+  async validateEmail(_email: string): Promise<false | MailgunValidateResult> {
```

Comment:
-1 The return type is not very clear for the caller of this function. I think we should just have a boolean send/do not send and leave the interpretation of the different responses to the mail manager implementations.
If you think that's not enough information, we could have a separate type here in the interface
```ts
export type ValidationResult =
  | 'valid'
  | 'invalid'
  | 'risky'
  | 'unknown'
```

Reply:
Sounds good! I've updated the return type to a boolean which fixes the other comments too